src/Common/Integrations/Plugin_Merge_Provider_Abstract.php

```diff
@@ -221,12 +223,13 @@ public function init(): void {
 			return;
 		}
 
-		// If the Event Ticket Wallet Plus plugin is active, we need to deactivate it before continuing to avoid a fatal.
+		// If the child plugin is active, we need to deactivate it before continuing to avoid a fatal.
 		if ( $this->is_child_plugin_active() ) {
 			$this->deactivate_plugin();
 
 			// Leave a notice of the forced deactivation.
 			$this->send_updated_merge_notice();
+			$this->send_updated_notice();
```

Comment:
I got confused about scenarios and removed this when I shouldnt have. The bug was the memoize.